recipes/cppad/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cppad" %}
+{% set version = "20210000.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/coin-or/CppAD/archive/{{ version }}.tar.gz
+  sha256: 6fae54ea670f07138a45a0aa6e0e4078999bbe7309eec3916ff892f0e81e9e4a
+  patches:
+    - include_cctype.patch
+
+build:
+  number: 0
+  # No ABI policy documented, let's be conservative 
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [linux]
+   
+test:
+  commands:
```

Comment:
Can you test for the existence of the shared library here or is this header-only?

Reply:
I added the tests in https://github.com/conda-forge/staged-recipes/pull/13628/commits/5e92cf08723cbea0f6a936367274666d586f26cd . For now on Windows the static library is compiled as the use of static libraries is supported in the upstream project, but I plan to eventually patch the upstream project and import the patch to generate shared libraries also on Windows, for [CFEP-18 compliance](https://github.com/conda-forge/cfep/blob/master/cfep-18.md).